pwndbg/gdblib/vmmap.py

```diff
@@ -453,9 +480,12 @@ def info_proc_maps(parse_flags=False) -> Tuple[pwndbg.lib.memory.Page, ...]:
         # On qemu user emulation, we may get: gdb.error: Not supported on this target.
         info_proc_mappings = []
 
+    # See if "Perms" is in the header line
+    perms_available = "Perms" in info_proc_mappings[3]
```

Comment:
Looking at the source code for `info proc maps` - Which by looking at https://github.com/bminor/binutils-gdb/blob/28c3bf55f0f9aca8619c6d01be34a02a887c5577/gdb/linux-tdep.c#L814, I made sure that it's always the fourth line of output.